### ISSUE TITLE
Compiling Model fix

### ISSUE THREAD

**Issue by Vishv07** [kothariji/SyntaxMeets#31]
Can we remove this
![image](https://user-images.githubusercontent.com/37216460/110306828-46352c80-8024-11eb-9213-95246a862a2c.png)

 🔨  It looks weird. we can keep it simple by just Placing loading in the Run Button while it's Executing the Code.

**Comment by kothariji** [comment]
Yes, we can think over it. Can you show some examples so that it would be more clear 😊

**Comment by Vishv07** [comment]
I want to work on this issue as a part of GSSOC'21

**Comment by kothariji** [comment]
@Vishv07 Great 🚀
I am assigning this issue to you 👍
